README.md

```diff
@@ -28,6 +28,13 @@
 This repository contains idiomatic C++ client libraries for the following
 [Google Cloud Platform](https://cloud.google.com/) services.
 
+> Please check the [CHANGELOG] for important announcements and upcoming
+> changes.<br>
+> In particular, note that on (or about) **2022-06-01** this library will stop
+> development based on C++11.
```

Comment:
We may want to make it clear that the new baseline will be C++14, which may not be obvious. What about saying "... this library will start requiring >= C++14"?

Reply:
Moved earlier in the text.

Reply:
JFYI, I don't see the fix, but I'm also happy to consider my comment optional if you'd prefer to leave the text as it is now.